android/pytorch_android/src/androidTest/java/org/pytorch/PytorchInstrumentedTests.java

```diff
@@ -298,6 +299,105 @@ public void testStr3Concat() throws IOException {
     }
   }
 
+  @Test
+  public void testAdd1ForByteTensors() throws IOException {
+    final long[] inputTensorShape = new long[]{256};
+    final long numElements = Tensor.numel(inputTensorShape);
+    final byte[] inputTensorData = new byte[(int) numElements];
+    for (int i = 0; i < numElements; ++i) {
+      inputTensorData[i] = (byte) i;
+    }
+    final Module module = Module.load(assetFilePath(TEST_MODULE_ASSET_NAME));
+
+    final Tensor inputTensorUint8 = Tensor.newUInt8Tensor(inputTensorShape, inputTensorData);
+    final Tensor inputTensorInt8 = Tensor.newInt8Tensor(inputTensorShape, inputTensorData);
+
+    final IValue inputUint8 = IValue.tensor(inputTensorUint8);
+    final IValue inputInt8 = IValue.tensor(inputTensorInt8);
+
+    final IValue outputUint8 = module.runMethod("tensorAdd1", inputUint8);
+    final Tensor outputTensorUint8 = outputUint8.getTensor();
+
+    final IValue outputInt8 = module.runMethod("tensorAdd1", inputInt8);
+    final Tensor outputTensorInt8 = outputInt8.getTensor();
+
+    byte[] outputDataUint8 = outputTensorUint8.getDataAsUnsignedByteArray();
+    byte[] outputDataInt8 = outputTensorInt8.getDataAsByteArray();
+
+    for (int i = 0; i < numElements; i++) {
+      final byte iB = (byte) i;
+      final byte outUint8B = outputDataUint8[i];
+      final byte outInt8B = outputDataInt8[i];
+
+      final int outUint8UI = byteToUnsignedInt(outUint8B);
+      final int expectedOutUint8UI = i != 255 ? i + 1 : 0;
+      final byte expectedOutUint8B = (byte) expectedOutUint8UI;
+      assertEqualsBytes(expectedOutUint8B, outUint8B);
+      assertEquals(expectedOutUint8UI, outUint8UI);
+
+      final int expectedOutInt8I = i == Byte.MAX_VALUE ? Byte.MIN_VALUE : ((int) iB) + 1;
+      final byte expectedOutInt8B = (byte) expectedOutInt8I;
+      assertEqualsBytes(expectedOutInt8B, outInt8B);
+      assertEquals(expectedOutInt8I, (int) outInt8B);
+    }
+  }
+
+  private static int byteToUnsignedInt(byte b) {
+    return ((int) b) & 0xff;
```

Comment:
Should be consistent with `0xff` or `0xFF`.  (I prefer `0xFF`, but it's a matter of taste).

Reply:
Also, I don't think the cast is necessary.  All byte and char are converted to int before math operations.